data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -658,7 +658,8 @@ spec:
                     hostPrefix:
                       description: HostPrefix is the prefix size to allocate to each
                         node from the CIDR. For example, 24 would allocate 2^8=256
-                        adresses to each node.
+                        adresses to each node. If this field is not used by the plugin,
+                        it can be unset(0).
```

Comment:
if you're going to keep "unset(0)" you need a space before the paren, but I think you can just say "it can be left unset"; if the user does not specify it in the input, then it will be 0 in the code, but the user doesn't really need to know that.

Reply:
> if you're going to keep "unset(0)" you need a space before the paren, but I think you can just say "it can be left unset"; 

I can change this to `it can be left unset` but..

> if the user does not specify it in the input, then it will be 0 in the code, but the user doesn't really need to know that.

I thought if the user does not specify it in the input the default values will take over in the installer ? https://github.com/openshift/installer/blob/7737b2a6f42740b214d2b2e10ba6abd6742a4b65/pkg/types/defaults/installconfig.go#L21

but I guess I am wrong because the default value 23 will take effect only when no cidr (or more precisely not clusternetwork) is specified at all.